src/services/vehicles.ts

```diff
@@ -0,0 +1,43 @@
+import { Vehicle } from '../models';
+import { VehicleInfo } from '../models/vehicle-info';
+import { FuelInfo } from '../models/fuel-info';
+
+export class Vehicles {
+    public static async insert(vehicle: VehicleInfo, fuelInfo: FuelInfo): Promise<number> {
+        const existingVehicle = await Vehicle.findOne({
+            where: { license: vehicle.kenteken },
+        });
+
+        if (existingVehicle) {
+            return existingVehicle.id;
```

Comment:
Please remove this obsolete else statement. It causes unnecessary indenting.

Reply:
ops, fixed